app/site_settings.py

```diff
@@ -11,4 +25,15 @@
 DEFAULT_LANGUAGE_CODE)
 """
 
-OPTIONAL_PATH_PREFIX = 'personfinder/'
+# When Person Finder is run at an address under a subdirectory rather than at
+# root (e.g., www.organization.org/personfinder/), that subdirectory should be
+# specified here so that URLs can be handled correctly. The value MUST NOT start
+# or end with a slash.
+OPTIONAL_PATH_PREFIX = 'personfinder'
+
+# Hosts allowed in prod (not applicable to local servers).
+PROD_ALLOWED_HOSTS = [
```

Comment:
It's a security measure ([here's the blog entry where they explained it](https://www.djangoproject.com/weblog/2013/feb/19/security/#s-issue-host-header-poisoning), and there's some more detail in [the settings docs](https://docs.djangoproject.com/en/1.11/ref/settings/#allowed-hosts)). The setting is required, so we have to put something there. I've set it to a wildcard for now (seems ok since afaik we/webapp2 weren't doing any host header validation before), but I put in a TODO to try to set it for real later.

Reply:
I see, thanks for the explanation!